sdk/arm/msi/2018-11-30/armmsi/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/Azure/azure-sdk-for-go/sdk/arm/msi/2018-11-30/armmsi
```

Comment:
The `go.mod` and `go.sum` files should go in the `./sdk/arm/msi` directory.  This will also cause the module name to change accordingly.

Reply:
I have moved the `go.mod` file